packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -752,23 +753,26 @@ async function triggerDownloadDocument({
     npi: facilityNPI,
     cxId,
   };
-  const lambdaResult = await lambdaClient
+  const result = await lambdaClient
     .invoke({
-      FunctionName: lambdaName,
+      FunctionName: docDownloaderLambdaName,
       InvocationType: "RequestResponse",
       Payload: JSON.stringify(payload),
     })
     .promise();
-
-  if (lambdaResult.StatusCode !== 200)
-    throw new MetriportError("Lambda invocation failed", undefined, { lambdaName, docId: doc.id });
-
-  if (lambdaResult.Payload === undefined)
-    throw new MetriportError("Payload is undefined", undefined, { lambdaName, docId: doc.id });
-
-  const newFile = JSON.parse(lambdaResult.Payload.toString());
-
-  return newFile;
+  //
+  // This should show up on a conflict w/ this PR https://github.com/metriport/metriport/pull/1013
+  // Need to make sure the cloud implentation of the doc downloader uses the new logic to handle lambda errors
+  //
+  // This should show up on a conflict w/ this PR https://github.com/metriport/metriport/pull/1013
+  // Need to make sure the cloud implentation of the doc downloader uses the new logic to handle lambda errors
+  //
+  // This should show up on a conflict w/ this PR https://github.com/metriport/metriport/pull/1013
+  // Need to make sure the cloud implentation of the doc downloader uses the new logic to handle lambda errors
+  //
```

Comment:
does the release plan need to be updated to take this into account?

Reply:
I'll have to fix this either here or on the PR that makes introduces the local impl, depending on which comes first.

I can choose one if both are g2g and let you know about the merge fix prior to merging it